localstripe/resources.py

```diff
@@ -817,6 +817,25 @@ def _api_create(cls, source=None, **data):
 
         return obj
 
+    @classmethod
+    def _api_list_all(cls, url, email=None, limit=None, starting_after=None,
```

Comment:
Can you put this function lower next to other `_api_list*` functions?

Reply:
Yes, of course.